api/handlers.go

```diff
@@ -552,6 +552,14 @@ func getEvents(c *context, w http.ResponseWriter, r *http.Request) {
 	c.eventsHandler.Wait(r.RemoteAddr, until)
 }
 
+// POST /exec/{execid:.*}/start
+func postExecStart(c *context, w http.ResponseWriter, r *http.Request) {
+	if r.Header.Get("Connection") == "" {
```

Comment:
What does the connection header imply? Is this something that would be usable anywhere else?